integration_tests/e2e/goal/updateGoal.cy.ts

```diff
@@ -90,6 +91,10 @@ context('Update a goal', () => {
       .setStepTargetDateRange(3, '6 to 12 months')
       .submitPage()
 
+    // Then
```

Comment:
Minor, but we dont want 2 `// Then` blocks, plus I think the code in your new `Then` block is actually part of the `When` 
My thinking is that in context of the test (adding a step as part of adding a goal) we need to go via the new Review Goal page in order to actually save the Goal (and its new step) - so it feels like its part of the `When` ?

How about this?
```
    // When
    updateGoalPage //
      // The goal loaded from the wiremock stub data has 2 steps. After adding another step, the new step will be referenced as step 3
      .addAnotherStep()
      .setStepTitle(3, 'A brand new step')
      .setStepStatus(3, 'Started')
      .setStepTargetDateRange(3, '6 to 12 months')
      .submitPage()
    const reviewUpdateGoalPage = Page.verifyOnPage(ReviewUpdateGoalPage)
    reviewUpdateGoalPage.submitPage()

    // Then
    Page.verifyOnPage(OverviewPage)
    // Assert that the expected ...
```

Reply:
Removed the duplicated `// Then` blocks